spec/install/gems/standalone_spec.rb

```diff
@@ -283,9 +283,16 @@
       end
     end
 
+    it "creates stubs that can be symlinked" do
+      require "tmpdir"
+      symlink = File.expand_path("rails", Dir.mktmpdir)
```

Comment:
`symlink = tmp("symlink", "rails")` to ensure the file is under bundlers control